### ISSUE TITLE
What does MLP module exactly optimize? 

### ISSUE THREAD

**Issue by donsmj** [keras-team/autokeras#543]
In MLP module, after running MlpModule.fit function, some models are obtained based on Loss and Metric value. I just want to know what does MLP module exactly optimize? Does it optimize architecture or hyperparameters?

highly appreciate if someone could address this to me because I am writing a report based on that.

Regards,
Mohammad
